src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/Generated/StorageAdminManagementClient.cs

```diff
@@ -13168,20 +13168,17 @@ public static Task<MetricsResult> GetMetricsAsync(this IShareOperations operatio
         /// <param name='farmId'>
         /// Required. Your documentation here.
         /// </param>
-        /// <param name='shareName'>
-        /// Required. Your documentation here.
```

Comment:
These should be filled in with actual documentation.  Also, Please provide a link to the PR in hydra-specs-pr that contaisn the corresponding specification change for this PR.